static/js/settings_bots.js

```diff
@@ -110,12 +110,8 @@ exports.generate_zuliprc_content = function (email, api_key) {
            "\n";
 };
 
-function bot_name_from_email(email) {
-    return email.substring(0, email.indexOf("-bot@"));
-}
-
 exports.generate_flaskbotrc_content = function (email, api_key) {
-    return "[" + bot_name_from_email(email) + "]" +
+    return "[]" +
```

Comment:
Why not do `[api]` here?

Reply:
I thought the plan was to keep the `[api]` in the `zuliprc`s, and use empty headers for the flaskbotrc?  For running single bots with a botserver, people will just use a `zuliprc`, so the flaskbotrc doesn't matter for that case. For running multiple bots, it doesn't matter if the downloaded flaskbotrc is invalid - we expect and want the user to edit it anyway. In such a case, `[api]` seems confusing to me, since it doesn't sound like it has something to do with what we actually want the user to write there - the bot they want to run.